tile-group.lisp

```diff
@@ -930,9 +930,46 @@ windows used to draw the numbers in. The caller must destroy them."
 "Split the current frame into 2 frames, one on top of the other."
   (split-frame-in-dir (current-group) :row (read-from-string ratio)))
 
-(defcommand (remove-split tile-group) (&optional (group (current-group)) (frame (tile-group-current-frame group))) ()
-"Remove the specified frame in the specified group (defaults to current
-group, current frame). Windows in the frame are migrated to the frame taking up its
+(defun split-frame-eql-parts* (group dir amt)
+  (when (not (eq amt 1))
```

Comment:
Compare numbers with `=`. In this case, use `/=` instead of `(not (= ...))`.  I'd actually think about using `>`.